### ISSUE TITLE
[Noetic] rtctree fails to build on all platforms

### ISSUE THREAD

**Issue by sloretz** [tork-a/rtctree-release#3]
* https://build.ros.org/view/Nbin_uF64/job/Nbin_uF64__rtctree__ubuntu_focal_amd64__binary/94/console
* https://build.ros.org/view/Nbin_ufhf_uFhf/job/Nbin_ufhf_uFhf__rtctree__ubuntu_focal_armhf__binary/94/console
* https://build.ros.org/view/Nbin_ufv8_uFv8/job/Nbin_ufv8_uFv8__rtctree__ubuntu_focal_arm64__binary/94/console

```
23:14:56 cd /tmp/binarydeb/ros-noetic-rtctree-3.0.1 && python setup.py build
23:14:56 /bin/sh: 1: python: not found
```

I opened it up here instead of https://github.com/gbiggs/rtctree/ as the upstream repo seems to be unmaintained.

**Comment by k-okada** [comment]
@sloretz thanks for heads up, I have fixed(https://github.com/tork-a/rtctree-release/blob/release/noetic/rtctree/CMakeLists.txt#L11 )  and released again ros/rosdistro#39462

**Comment by k-okada** [comment]
closed via https://github.com/ros/rosdistro/pull/39462
